contracts/governance/utils/Votes.sol

```diff
@@ -225,14 +225,6 @@ abstract contract Votes is Context, EIP712, Nonces, IERC5805 {
         return a - b;
     }
 
-    /**
-     * @dev Returns the contract's {EIP712} domain separator.
-     */
-    // solhint-disable-next-line func-name-mixedcase
-    function DOMAIN_SEPARATOR() external view returns (bytes32) {
```

Comment:
I think this change justifies a changeset.

Reply:
I added a note to the existing changeset where we mention that ERC20Votes is no longer implicitly ERC20Permit.